core/DependencyInjection/ContainerBuilder.php

```diff
@@ -183,7 +185,7 @@ private function getTemporaryServiceFileContent(): string
         
             %s
         };',
-            var_export($this->middlewareExtensionsMapper->map($extensions), true),
+            var_export($this->middlewareExtensionsMapper->map(...$extensions), true),
```

Comment:
Suggestion: It may be better to use `serialize` instead of `var_export` here?

Reply:
I can't, cause I use the array keys to get this from the cache. Look the difference. 

```
php > echo serialize(['a' => 'b', 'c' => ['d']]);
a:2:{s:1:"a";s:1:"b";s:1:"c";a:1:{i:0;s:1:"d";}}
php > var_export(['a' => 'b', 'c' => ['d']]);
array (
  'a' => 'b',
  'c' => 
  array (
    0 => 'd',
  ),
)
php >
```

Reply:
You are right, marking this thread as solved :)